kb/plane_geometry/formal_statements/statements/statements_scs/stat_rectangular_triangle_cathetus_median_acute_angle_opposite.scs

```diff
@@ -0,0 +1,16 @@
+stat_rectangular_triangle_cathetus_median_acute_angle_opposite <- statement;;
+
+stat_rectangular_triangle_cathetus_median_acute_angle_opposite -> rrel_key_sc_element: concept_rectangular_triangle; nrel_cathetus; nrel_median; concept_acute_angle; nrel_opposite;;
+
+stat_rectangular_triangle_cathetus_median_acute_angle_opposite => 
+nrel_main_idtf: [Акс.(прямоугольный треугольник;катет*;медиана*;острый угол;противолежащий*)] (*<- lang_ru;;*);;
+stat_rectangular_triangle_cathetus_median_acute_angle_opposite => 
+nrel_main_idtf: [Ax.(rectangular triangle;cathetus*;median*;acute angle;opposite*)] (*<- lang_en;;*);;

```

Comment:
It isn't an axiom. It is a statement. You should use [Stat.(...)] insted of [Ax.(...)]